abe/unittest.py

```diff
@@ -69,7 +69,7 @@ def assert_data_dict_equal(self, data1, data2):
 
     def assert_data_list_equal(self, data1, data2):
         """
-        Two lists are recursively equal without taking order into account
+        Two lists are recursively equal, including ordering.
```

Comment:
Good point, ordering for dict fields is not relevant, but for lists it is, in particular when testing endpoints that do sorting.